src/Administration/Resources/app/administration/src/module/sw-mail-template/page/sw-mail-template-index/sw-mail-template-index.spec.js

```diff
@@ -72,4 +76,19 @@ describe('modules/sw-mail-template/page/sw-mail-template-index', () => {
 
         expect(createButton.attributes('disabled')).toBeUndefined();
     });
+
+    it('should update the route when searching mail templates', async () => {
+        const wrapper = await createWrapper();
+
+        const searchBar = wrapper.getComponent({ name: 'sw-search-bar' });
+        searchBar.vm.$emit('search', 'Invoice');
```

Comment:
```suggestion
        await searchBar.trigger('search', 'Invoice');
```
(simulates "real" dom event)

Reply:
Hm, "search" isn't a DOM event and the codebase is full of `vm.$emit()` to trigger Vue events. I'd need to modify the stub template to emit the search event via some native event here, which would test search bar internals rather than this component's handling of the search event